### ISSUE TITLE
Crash when offline

### ISSUE THREAD

**Issue by parrots** [mapbox/mapbox-events-ios#118]
This might be related to https://github.com/mapbox/mapbox-events-ios/issues/69, but I'm seeing crashes in production when users are offline after having built with Xcode 10.2. This is with version 0.9.1.

 Specifically:
```
*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[3]
+[MMEEvent debugEventWithError:]

Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x1abe8dea4 __exceptionPreprocess
1  libobjc.A.dylib                0x1ab05da50 objc_exception_throw
2  CoreFoundation                 0x1abe05384 _CFArgv
3  CoreFoundation                 0x1abd8b870 -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]
4  CoreFoundation                 0x1abd7c3b4 +[NSDictionary dictionaryWithObjects:forKeys:count:]
5  MapboxMobileEvents             0x105e79d78 +[MMEEvent debugEventWithError:] (MMEEvent.m:163)
6  MapboxMobileEvents             0x105e7ff94 __52-[MMEEventsManager sendPendingTelemetryMetricsEvent]_block_invoke (MMEEventsManager.m:362)
7  MapboxMobileEvents             0x105e75458 __45-[MMEAPIClient postEvents:completionHandler:]_block_invoke (MMEAPIClient.m:99)
8  libdispatch.dylib              0x1ab8c56c8 _dispatch_call_block_and_release
9  libdispatch.dylib              0x1ab8c6484 _dispatch_client_callout
10 libdispatch.dylib              0x1ab8a5b34 _dispatch_main_queue_callback_4CF$VARIANT$armv81
11 CoreFoundation                 0x1abe1dce4 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
12 CoreFoundation                 0x1abe18bac __CFRunLoopRun
13 CoreFoundation                 0x1abe180e0 CFRunLoopRunSpecific
14 GraphicsServices               0x1ae091584 GSEventRunModal
15 UIKitCore                      0x1d9068c00 UIApplicationMain
16 Slopes                         0x1046558dc main (main.swift:12)
17 libdyld.dylib                  0x1ab8d6bb4 start
```

**Comment by rclee** [comment]
Thanks for the report @parrots! We'll get this resolved soon, sorry for any inconvenience 🙇

**Comment by alfwatt** [comment]
Fix in: https://github.com/mapbox/mapbox-events-ios/pull/119

**Comment by alfwatt** [comment]
Fixed in 0.9.2 pending integration
